src/hotspot/share/gc/parallel/mutableSpace.hpp

```diff
@@ -58,7 +59,7 @@ class MutableSpace: public CHeapObj<mtGC> {
   size_t _alignment;
   HeapWord* _bottom;
   HeapWord* volatile _top;
-  HeapWord* _end;
+  HeapWord* volatile _end;
```

Comment:
Maybe add some comments explaining how `_top` and `_end` are used in the concurrent setting.

Reply:
I've added some comments describing `_bottom`, `_top`, and `_end`.